src/Mintette/MintetteOptions.hs

```diff
@@ -34,6 +37,13 @@ data ServeOptions = ServeOptions
     , cloActionLogsLimit :: Word
     }
 
+data AddToBankOptions = AddToBankOptions
+    { cloMintetteHost     :: String
+    , cloMintettePort     :: Int
+    , cloSecretKeyPath_   :: FilePath
+    , cloAutoCreateKey_   :: Bool
```

Comment:
Does it make sense to have `auto create key` here? In my opinion, the only use case is that person create keypair first, then sends it to *Bank* somehow and after some time uses this command with already existing key.